src/components/LoansByCategory/LoansBundleExpWrapper.vue

```diff
@@ -80,6 +80,7 @@
 import numeral from 'numeral';
 import updateLoanReservation from '@/graphql/mutation/updateLoanReservation.graphql';
 import KivaClassicLoanCarouselExp from '@/components/LoanCollections/KivaClassicLoanCarouselExp';
+import LoanCardController from '@/components/LoanCards/LoanCardController';
```

Comment:
yep this is important ;)

Reply:
Yeah! 
We are having a new component, LoansBundleExpWrapper, for bundling, it will be easier to maintain and implement in other pages.